lib/tasks/deploy.rake

```diff
@@ -2,6 +2,7 @@ GITHUB_ORG = 'ad2games'
 BASE_IMAGE = 'docker-rails:latest'
 DEPLOY_USER = 'ad2gamesdeploy'
 DEPLOY_EMAIL = 'developers@ad2games.com'
+DEPLOY_ENVS = { 'staging' => %w(staging) } # N.b., master is not deployed by default
```

Comment:
In some cases we only have `preview` but no `staging` (adserver, exchange, topgames). So we need to address that as well. Maybe we just make the DEPLOY_ENVS param mandatory?